server/chat-plugins/room-events.js

```diff
@@ -9,6 +9,18 @@
 
 'use strict';
 
+let formatEvent = function(event) {
+	let timeRemaining = new Date(event.date).getTime() - new Date().getTime();
+	if (!timeRemaining) timeRemaining = "The time remaining for this event is not available";
+	if (timeRemaining < 0) timeRemaining = "This event will start soon";
+	if (event.started) timeRemaining = "This event has started";
+	let ret = `<tr title="${isNaN(timeRemaining) ? timeRemaining : `This event will start in: ${Chat.toDurationString(timeRemaining, {precision: 2})}`}">`;
```

Comment:
Nested template strings aren't very readable. If you're going to split up the string buffer into multiple lines away, construct the string the line before.

Reply:
Done